doc/api/cli.md

```diff
@@ -340,6 +358,16 @@ misformatted, but any errors are otherwise ignored.
 Note that neither the well known nor extra certificates are used when the `ca`
 options property is explicitly specified for a TLS or HTTPS client or server.
 
+### `SSL_CERT_DIR=dir`
+
+If `--use-openssl-ca` is enabled, this overrides and sets OpenSSL's directory
+containing trusted certificates.
+
+### `SSL_CERT_FILE=file`
```

Comment:
@jasnell this is out of our control, the env vars come as part of the OpenSSl trust store feature. This env var is implemented by OpenSSL in https://github.com/nodejs/node/blob/master/deps/openssl/openssl/crypto/x509/by_file.c#L100, which also implies its an env var supported by pretty much every program that uses OpenSSL, unless they have their own built-in trust store (Node is unusual in this respect).